kvbc/include/merkle_tree_serialization.h

```diff
@@ -105,6 +111,7 @@ inline std::string serializeImp(const sparse_merkle::Hash &hash) {
 }
 
 inline void serializeImp(const sparse_merkle::InternalNodeKey &key, std::string &out) {
+  serializeImp(key.address(), out);
```

Comment:
Is it possible to have a prefix to distinguish for other types of nodes ? (we would have all the Merkle keys in the same CF).